db/primary_migrate/20210809203128_drop_good_jobs_main_database.rb

```diff
@@ -0,0 +1,24 @@
+class DropGoodJobsMainDatabase < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :good_jobs, id: :uuid do |t|
+      t.text :queue_name
+      t.integer :priority
+      t.jsonb :serialized_params
+      t.timestamp :scheduled_at
+      t.timestamp :performed_at
+      t.timestamp :finished_at
+      t.text :error
```

Comment:
This is an aggressive migration... it just drops the jobs table in the old/main DB.

I think this is safe to deploy so long as we get the separate DB up in prod first before deploying

Reply:
Update, I'm moving this into a separate PR for extra safety